examples/tv-casting-app/APIs.md

```diff
@@ -1,123 +1,155 @@
 
 ## Introduction
-The Matter Casting APIs are consumed by a Casting client like an Android / iOS app that may want to cast videos, music, pictures or some other content on a `CastingPlayer`. The Casting client is expected to be a Matter Commissionable Node and a `CastingPlayer` is expected to be a Matter Commissioner. In the context of the [Matter Video Player architecture](https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/app_clusters/media/VideoPlayerArchitecture.adoc), a `CastingPlayer` would map to [Casting "Video" Player](https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/app_clusters/media/VideoPlayerArchitecture.adoc#1-introduction). The `CastingPlayer` is expected to be hosting one or more `Endpoints` (similar to [Content Apps](https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/app_clusters/media/VideoPlayerArchitecture.adoc#1-introduction) in the Matter Video Player architecture) that support one or more Matter Media `Clusters`. 
-The Casting client can use the Matter Casting APIs to **discover** CastingPlayers, **connect** to them and **interact** with one or more of their Endpoints i.e send Cluster commands and read / subscribe to attributes, to realize an End-to-End Casting experience.
+
+A Casting client (e.g. a mobile phone app) is expected to be a Matter Commissionable Node and a `CastingPlayer` (i.e. a TV) is expected to be a Matter Commissioner. In the context of the [Matter Video Player architecture](https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/app_clusters/media/VideoPlayerArchitecture.adoc), a `CastingPlayer` would map to [Casting "Video" Player](https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/app_clusters/media/VideoPlayerArchitecture.adoc#1-introduction). The `CastingPlayer` is expected to be hosting one or more `Endpoints` (similar to [Content Apps](https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/app_clusters/media/VideoPlayerArchitecture.adoc#1-introduction) in the Matter Video Player architecture) that support one or more Matter Media `Clusters`.
+
+The steps to start a casting session are:
+
+1. [Initialize](#initialize-the-casting-client) the `CastingClient` using the Matter SDK.
+1. [Discover](#discover-casting-players) `CastingPlayer` devices using Matter Commissioner discovery.
+1. [Connect](#connect-to-a-casting-player) to the `CastingPlayer` to discover available endpoints.
+1. [Select](#select-an-endpoint-on-the-casting-player) an available `Endpoint` hosted by the `CastingPlayer`.
+
+Next, you're ready to:
+
+1. [Issue commands](#issuing-commands) to the `Endpoint`.
+1. [Read](#read-operations) playback state.
```

Comment:
This "playback state" feels like a fuzzy term, given that there is a specific attribute in the MediaPlayback cluster called CurrentState. But a Casting client may be interested in reading many different attributes beyond this one.

Reply:
I'm open to suggestions here... maybe we can say "Read endpoint attributes like playback state"?

Reply:
Yes, that'll make more sense!